README.md

```diff
@@ -18,7 +18,11 @@ The web server repository can be found [here](https://github.com/marcelorcorrea/
 
 Feel free to fork or contribute this project.
 
-Download in [Play Store](https://play.google.com/store/apps/details?id=org.falaeapp.falae)
+[![Slack channel](https://img.shields.io/badge/Chat-Slack-blue.svg)](https://falaeapp.slack.com)
+[![Language Kotlin](https://img.shields.io/badge/Language-Kotlin-blue.svg)](https://kotlinlang.org/)
+[![License MIT](https://img.shields.io/badge/Licence-Mit-blue.svg)](LICENSE)
+[![Downolad Android](https://img.shields.io/badge/Download-Android-blue.svg)](https://play.google.com/store/apps/details?id=org.falaeapp.falae)
```

Comment:
Nit: `Downolad`

Reply:
yeah, but this is pretty stable, I saw many power git pages using it. Anyway, it will not crash our app if this failed in some time.